src/routes/disposal-recovery-codes.test.js

```diff
@@ -60,7 +60,7 @@ describe('Create Receipt Movement - Disposal/Recovery Code Validation', () => {
                 code,
                 weight: {
                   metric: 'Tonnes',
-                  amount: 0.1,
+                  amount: 1,
```

Comment:
decimal values are valid for the amount property, so shouldn't be needed to change this test data